wordpress-mobile/WordPress-iOS
# Plugin updates

Adds support to update a plugin.

![plugin-updates](https://user-images.githubusercontent.com/8739/33763275-286c211a-dc10-11e7-82dc-a63ddced4fde.png)

![wpios-plugin-updates](https://user-images.githubusercontent.com/8739/33829166-6883b932-de6f-11e7-9ce1-dc560de4647f.gif)


Fixes #7471

To test:

- Make sure you have updates available for a plugin
- Go to the plugins list, the update icon should show
- Go to the plugin detail, it should say what version is available
- Tap the update button, the button should say "Updating", and if you go back to the list, the update icon should animate.
- When the update finishes, the update icon in the list should disappear, and the "version available" row should be gone from the plugin detail view.
